### PR TITLE
Description for encrypt/decrypt feature

### PR DESCRIPTION
**Pull request by logvik**
https://github.com/MetaMask/metamask-extension/pull/7831

### CODE REVIEW

File: docs/guide/experimental-apis.md [hunk 1]
```diff
@@ -57,3 +57,50 @@ This method returns a `Promise` that resolves to a `Boolean` indicating if the c
...
+
+### eth_getEncryptionPublicKey
+
+This method is used for getting a public key that based on the `nacl` [library](https://github.com/dchest/tweetnacl-js). With such a key, you can encrypt a message.
+
+```javascript
+ethereum.sendAsync({
+    jsonrpc: '2.0',
+    method: 'eth_getEncryptionPublicKey',
+    params: [web3.eth.defaultAccount],
```
Comment (jennypollack) [inline docs/guide/experimental-apis.md]:
```suggestion
    params: [accounts[0]],
```

File: docs/guide/experimental-apis.md [hunk 2]
```diff
@@ -57,3 +57,50 @@ This method returns a `Promise` that resolves to a `Boolean` indicating if the c
...
+    from: web3.eth.defaultAccount,
```
Comment (jennypollack) [inline docs/guide/experimental-apis.md]:
```suggestion
    from: accounts[0],
```

File: docs/guide/experimental-apis.md [hunk 3]
```diff
@@ -57,3 +57,50 @@ This method returns a `Promise` that resolves to a `Boolean` indicating if the c
...
+
+### eth_decrypt
+
+This method is used to decrypt a message that was encrypted using the public key based on the `nacl` [library](https://github.com/dchest/tweetnacl-js).
+
+```javascript
+web3.currentProvider.sendAsync({
+    jsonrpc: '2.0',
+    method: 'eth_decrypt',
+    params: [encryptedMessage, web3.eth.defaultAccount],
```
Comment (jennypollack) [inline docs/guide/experimental-apis.md]:
```suggestion
    params: [encryptedMessage, accounts[0]],
```

File: docs/guide/experimental-apis.md [hunk 4]
```diff
@@ -57,3 +57,50 @@ This method returns a `Promise` that resolves to a `Boolean` indicating if the c
...
+    from: web3.eth.defaultAccount,
```
Comment (jennypollack) [inline docs/guide/experimental-apis.md]:
```suggestion
    from: accounts[0],
```

**Comment by nicc938** [comment]
Thxx

**Pull request merged**